docs/source/WritingTests.rst

```diff
@@ -1197,174 +1197,70 @@ or not, please refer to our section on :ref:`docstring-directive-rules`.
 
 Now let's follow with some docstring directives examples.
 
-.. _docstring-directive-enable-disable:
+.. _docstring-directive-recursive:
 
-Explicitly enabling or disabling tests
---------------------------------------
+Test Inheritance
```

Comment:
Well this whole chapter is quite unnecessary as users perhaps don't need those details. I mean currently it works as one would expect from eg python unittests. How about to describe where we diverge, one paragraph per each subject:

1. We use static analysis to find tests
2. Inherited tests are discovered + the example you have here (it's nice)
3. Dynamic `test*` methods generation is not possible (it's important to mention this)
4. Disabling tests (which is now a separate section, but a paragraph should perhaps be enough).

I think it'd be shorter and easier to digest. (but I might be wrong)

Reply:
Btw what made me think about this was how that test promotes how Avocado is great that it can also discover inherited tests... That is normal, not extraordinary. Btw thinking about it twice I don't even think the example is necessary. People should just write the code and it should work. It makes sense to mention that as part of explanation that we don't load the modules, but only saying tests can be inherited from other tests should be fine. Important is to state we don't allow dynamic test* method generation as that is where we digress from "usual".